### ISSUE TITLE
Meteor works to meteor doesn't work in a few steps

### ISSUE THREAD

**Issue by QbyteFennek** [meteor/meteor#7844]
Starting from fresh install

```
C:\Users\jente\meteor>git clone https://github.com/bsliran/angular2-meteor-base.
Cloning into 'socially'...
remote: Counting objects: 387, done.
remote: Compressing objects: 100% (12/12), done.
Receivingotal 387 (delta 4), reused 0 (delta 0), pack-reused 375R
Receiving objects: 100% (387/387), 74.93 KiB | 0 bytes/s, done.
Resolving deltas: 100% (163/163), done.
```

```
C:\Users\jente\meteor>cd socially
```

```
C:\Users\jente\meteor\socially>meteor npm install

> angular2-meteor-base@ postinstall C:\Users\jente\meteor\socially
> typings i


+-- chai-spies@0.7.1
| `-- chai
|   `-- assertion-error
`-- meteor@1.3 (global)
```

```
C:\Users\jente\meteor\socially>meteor --port 4000
[[[[[ C:\Users\jente\meteor\socially ]]]]]

=> Started proxy.
=> Started MongoDB.
=> Errors prevented startup:

   While checking for angular2-compilers@0.6.1:
   error: No compatible binary build found for this package. Contact the package

=> Your application has errors. Waiting for file change.
Terminate batch job (Y/N)? y
```

```
C:\Users\jente\meteor\socially>cd ..
```

```
C:\Users\jente\meteor\socially>del socially
```

```
C:\Users\jente\meteor>git clone https://github.com/mschwarzmueller/angular-2-beta-boilerplate
Cloning into 'socially'...
remote: Counting objects: 137, done.
remote: Total 137 (delta 0), reused 0 (delta 0), pack-reused 137
Receiving objects: 100% (137/137), 26.40 KiB | 0 bytes/s, done.
Resolving deltas: 100% (56/56), done.
```

```
C:\Users\jente\meteor>git clone https://github.com/bsliran/angular2-meteor-base socially
Cloning into 'socially'...
remote: Counting objects: 387, done.
remote: Compressing objects: 100% (12/12), done.
Receivingotal 387 (delta 4), reused 0 (delta 0), pack-reused 375R
Receiving objects: 100% (387/387), 74.93 KiB | 0 bytes/s, done.
Resolving deltas: 100% (163/163), done.
```

```
C:\Users\jente\meteor>cd socially
```

```
C:\Users\jente\meteor\socially>meteor npm install
The system cannot find the path specified.
```

```
C:\Users\jente\meteor\socially>meteor
The system cannot find the path specified.
```

Tried reinstalling multiple times, always the same.

**Comment by abernix** [comment]
I have a lot of questions:

>    C:\Users\jente\meteor>git clone https://github.com/bsliran/angular2-meteor-base.

Why the trailing period?  This wouldn't work on my Git.

****

>    C:\Users\jente\meteor>git clone https://github.com/bsliran/angular2-meteor-base.
>    Cloning into 'socially'...

>    C:\Users\jente\meteor>git clone https://github.com/mschwarzmueller/angular-2-bet
>    Cloning into 'socially'...

Why do all of your `git clone` commands clone into the same `socially` directory?  They should be cloning into a directory with the name of the repo (e.g. `angular2-meteor-base` or `angular-2-bet` in this case)


****
>     C:\Users\jente\meteor\socially>cd ..
>     C:\Users\jente\meteor\socially>del socially

This doesn't make sense in your reproduction.  You didn't actually go down a folder, but you're trying to delete the `socially` folder still?

****
>     C:\Users\jente\meteor>git clone https://github.com/mschwarzmueller/angular-2-bet

This Git repo doesn't exist, but you're showing that you cloned it?

****

I'm guessing something is modifying your `PATH` during this process, though it's hard to tell.  You can check by running `SET` between each command.

**Comment by QbyteFennek** [comment]
@abernix 

About same directory: I removed the folder manually, my bad for not mentioning.
About non existent repo: Copy paste cut it off, fixed it.

About SET: Could you push me in the right direction of what I should be setting? There's a few different path variables for meteor.

**Comment by abernix** [comment]
You shouldn't have to set anything yourself, but it would seem that your `PATH` is getting changed.  I'm curious if you can print out the value of `PATH` before and after the problem.

If you're using Command Prompt you can do:

    C:\blah\blah> set PATH

and if you're using PowerShell, I think it's:

    C:\blah\blah> Get-ChildItem Env:PATH

It should contain something like:

    C:\Users\YourUsername\AppData\Local\.meteor\

Also, just as a reminder, make sure you restart Windows after you install Meteor, to ensure that the PATH is set properly.

**Comment by QbyteFennek** [comment]
@abernix 

Path has this: C:\Users\jente\AppData\Local\.meteor\
When I use 'meteor' as a run command it opens the parent folder of socially which is called meteor. Is this expected behavior? I guess not. Might the problem be that my parent folder of the meteor app is called meteor? Possibly it's tripping over this or do you think it's rather something different?

**Comment by abernix** [comment]
> When I use 'meteor' as a run command it opens the parent folder of socially

What do you mean by "opens"?

> Might the problem be that my parent folder of the meteor app is called meteor?

I don't think so but please do investigate!

****

Unfortunately, I don't have any ideas at the moment.  What version of Windows are you on and what Command Prompt are you using (e.g. PowerShell, `cmd` Command Prompt, etc.)?  I'm not on Windows regularly, but I will try to check the next time I boot it up.

**Comment by QbyteFennek** [comment]
@abernix I reinstalled clean and didn't use a folder named meteor and the problem doesn't appear anymore. Maybe there is some other action I'm not doing but I can't find it.

**Comment by abernix** [comment]
Hmm, okay.  I tried reproducing this using your steps above (but with a few changes since your steps are impossible otherwise).    No popup windows or 'The system cannot find' messages at all!

Here's the output:

```
Microsoft Windows [Version 10.0.10586]
(c) 2015 Microsoft Corporation. All rights reserved.

C:\Users\IEUser>npm install typings -g
C:\Users\IEUser\AppData\Roaming\npm\typings -> C:\Users\IEUser\AppData\Roaming\npm\node_modules\typings\dist\bin.js
typings@1.4.0 C:\Users\IEUser\AppData\Roaming\npm\node_modules\typings
├── has-unicode@2.0.1
├── listify@1.0.0
├── xtend@4.0.1
├── wordwrap@1.0.0
├── archy@1.0.0
├── minimist@1.2.0
├── any-promise@1.3.0
├── chalk@1.1.3 (escape-string-regexp@1.0.5, supports-color@2.0.0, ansi-styles@2.2.1, has-ansi@2.0.0, strip-ansi@3.0.1)
├── bluebird@3.4.6
├── columnify@1.5.4 (strip-ansi@3.0.1, wcwidth@1.0.1)
├── update-notifier@1.0.2 (is-npm@1.0.0, lazy-req@1.1.0, xdg-basedir@2.0.0, semver-diff@2.1.0, boxen@0.6.0, configstore@2.1.0, latest-version@2.0.0)
└── typings-core@1.6.0 (array-uniq@1.0.3, thenify@3.2.1, popsicle-rewrite@1.0.0, zip-object@0.1.0, popsicle-retry@3.2.1, popsicle-status@2.0.0, graceful-fs@4.1.9, promise-finally@2.2.1, throat@3.0.0, lockfile@1.0.2, sort-keys@1.1.2, string-template@1.0.0, strip-bom@2.0.0, make-error-cause@1.2.1, debug@2.2.0, has@1.0.1, rc@1.1.6, configstore@2.1.0, object.pick@1.1.2, parse-json@2.2.0, detect-indent@4.0.0, invariant@2.2.1, touch@1.0.0, mkdirp@0.5.1, is-absolute@0.2.5, popsicle-proxy-agent@3.0.0, rimraf@2.5.4, popsicle@8.2.0, typescript@2.0.3)

C:\Users\IEUser>mkdir meteor

C:\Users\IEUser>cd meteor

C:\Users\IEUser\meteor>git clone https://github.com/bsliran/angular2-meteor-base socially
Cloning into 'socially'...
remote: Counting objects: 397, done.
Receiving objects: 100% (397/397), 75.25 KiB | 0 bytes/s, done.7

Resolving deltas: 100% (172/172), done.
Checking connectivity... done.

C:\Users\IEUser\meteor>

C:\Users\IEUser\meteor>cd socially

C:\Users\IEUser\meteor\socially>meteor npm install

> angular2-meteor-base@ postinstall C:\Users\IEUser\meteor\socially
> typings i


+-- chai-spies@0.7.1
| `-- chai
|   `-- assertion-error
`-- meteor@1.3 (global)


C:\Users\IEUser\meteor\socially>meteor --port 4000
[[[[[ C:\Users\IEUser\meteor\socially ]]]]]

=> Started proxy.
=> Started MongoDB.
=> A patch (Meteor 1.4.1.2) for your current release is available!
   Update this project now with 'meteor update --patch'.
=> Errors prevented startup:

   While checking for angular2-compilers@0.6.1:
   error: No compatible binary build found for this package. Contact the package author and ask them to publish it
   for your platform.

=> Your application has errors. Waiting for file change.
Terminate batch job (Y/N)?
^C
C:\Users\IEUser\meteor\socially>

C:\Users\IEUser\meteor\socially>cd ..

C:\Users\IEUser\meteor>del /s /q socially
Deleted file - C:\Users\IEUser\meteor\socially\.gitignore
Deleted file - C:\Users\IEUser\meteor\socially\package.json
*** snipped for compactness ***
Deleted file - C:\Users\IEUser\meteor\socially\server\imports\server-main\main.test.ts
Deleted file - C:\Users\IEUser\meteor\socially\server\imports\server-main\main.ts

C:\Users\IEUser\meteor>rmdir /s /q socially

C:\Users\IEUser\meteor>git clone https://github.com/mschwarzmueller/angular-2-beta-boilerplate socially
Cloning into 'socially'...
remote: Counting objects: 137, done.
Receiving objects: 100% (137/137), 26.39 KiB | 0 bytes/s, done.7
Resolving deltas: 100% (56/56), done.

Checking connectivity... done.

C:\Users\IEUser\meteor>git clone https://github.com/bsliran/angular2-meteor-base socially
fatal: destination path 'socially' already exists and is not an empty directory.

*** I wasn't sure what else to do here but delete the dirs ***

C:\Users\IEUser\meteor>del /s /q socially

C:\Users\IEUser\meteor>rmdir /s /q socially

C:\Users\IEUser\meteor>git clone https://github.com/bsliran/angular2-meteor-base socially
Cloning into 'socially'...
remote: Counting objects: 397, done.
Receiving objects: 100% (397/397), 75.25 KiB | 0 bytes/s, done.7Receiving objects:  77% (306/397)

Resolving deltas: 100% (172/172), done.
Checking connectivity... done.

C:\Users\IEUser\meteor>

C:\Users\IEUser\meteor>cd socially

C:\Users\IEUser\meteor\socially>meteor npm install

> angular2-meteor-base@ postinstall C:\Users\IEUser\meteor\socially
> typings i


+-- chai-spies@0.7.1
| `-- chai
|   `-- assertion-error
`-- meteor@1.3 (global)

angular2-meteor-base@ C:\Users\IEUser\meteor\socially
+-- @angular/common@2.0.1
+-- @angular/compiler@2.0.1
+-- @angular/core@2.0.1
+-- @angular/forms@2.0.1
+-- @angular/platform-browser@2.0.1
+-- @angular/platform-browser-dynamic@2.0.1
+-- @angular/router@3.0.1
+-- @types/chai@3.4.34
+-- @types/mocha@2.2.32
+-- angular2-meteor@0.7.0
| `-- lodash@4.16.4
+-- angular2-meteor-polyfills@0.1.1
+-- angular2-meteor-tests-polyfills@0.0.2
+-- chai@3.5.0
| +-- assertion-error@1.0.2
| +-- deep-eql@0.1.3
| | `-- type-detect@0.1.1
| `-- type-detect@1.0.0
+-- chai-spies@0.7.1
+-- meteor-node-stubs@0.2.3
| +-- assert@1.3.0
| +-- browserify-zlib@0.1.4
| | `-- pako@0.2.8
| +-- buffer@4.5.1
| | +-- base64-js@1.1.2
| | +-- ieee754@1.1.6
| | `-- isarray@1.0.0
| +-- console-browserify@1.1.0
| | `-- date-now@0.1.4
| +-- constants-browserify@1.0.0
| +-- crypto-browserify@3.11.0
| | +-- browserify-cipher@1.0.0
| | | +-- browserify-aes@1.0.6
| | | | +-- buffer-xor@1.0.3
| | | | `-- cipher-base@1.0.2
| | | +-- browserify-des@1.0.0
| | | | +-- cipher-base@1.0.2
| | | | `-- des.js@1.0.0
| | | |   `-- minimalistic-assert@1.0.0
| | | `-- evp_bytestokey@1.0.0
| | +-- browserify-sign@4.0.0
| | | +-- bn.js@4.11.1
| | | +-- browserify-rsa@4.0.1
| | | +-- elliptic@6.2.3
| | | | +-- brorand@1.0.5
| | | | `-- hash.js@1.0.3
| | | `-- parse-asn1@5.0.0
| | |   +-- asn1.js@4.5.2
| | |   | `-- minimalistic-assert@1.0.0
| | |   +-- browserify-aes@1.0.6
| | |   | +-- buffer-xor@1.0.3
| | |   | `-- cipher-base@1.0.2
| | |   `-- evp_bytestokey@1.0.0
| | +-- create-ecdh@4.0.0
| | | +-- bn.js@4.11.1
| | | `-- elliptic@6.2.3
| | |   +-- brorand@1.0.5
| | |   `-- hash.js@1.0.3
| | +-- create-hash@1.1.2
| | | +-- cipher-base@1.0.2
| | | +-- ripemd160@1.0.1
| | | `-- sha.js@2.4.5
| | +-- create-hmac@1.1.4
| | +-- diffie-hellman@5.0.2
| | | +-- bn.js@4.11.1
| | | `-- miller-rabin@4.0.0
| | |   `-- brorand@1.0.5
| | +-- inherits@2.0.1
| | +-- pbkdf2@3.0.4
| | +-- public-encrypt@4.0.0
| | | +-- bn.js@4.11.1
| | | +-- browserify-rsa@4.0.1
| | | `-- parse-asn1@5.0.0
| | |   +-- asn1.js@4.5.2
| | |   | `-- minimalistic-assert@1.0.0
| | |   +-- browserify-aes@1.0.6
| | |   | +-- buffer-xor@1.0.3
| | |   | `-- cipher-base@1.0.2
| | |   `-- evp_bytestokey@1.0.0
| | `-- randombytes@2.0.3
| +-- domain-browser@1.1.7
| +-- events@1.1.0
| +-- http-browserify@1.7.0
| | +-- Base64@0.2.1
| | `-- inherits@2.0.1
| +-- https-browserify@0.0.1
| +-- os-browserify@0.2.1
| +-- path-browserify@0.0.0
| +-- process@0.11.2
| +-- punycode@1.4.1
| +-- querystring-es3@0.2.1
| +-- readable-stream@2.0.6
| | +-- core-util-is@1.0.2
| | +-- inherits@2.0.1
| | +-- isarray@1.0.0
| | +-- process-nextick-args@1.0.6
| | `-- util-deprecate@1.0.2
| +-- stream-browserify@2.0.1
| | `-- inherits@2.0.1
| +-- string_decoder@0.10.31
| +-- timers-browserify@1.4.2
| +-- tty-browserify@0.0.0
| +-- url@0.11.0
| | +-- punycode@1.3.2
| | `-- querystring@0.2.0
| +-- util@0.10.3
| | `-- inherits@2.0.1
| `-- vm-browserify@0.0.4
|   `-- indexof@0.0.1
+-- meteor-rxjs@0.3.0
+-- reflect-metadata@0.1.8
+-- rxjs@5.0.0-beta.12
| `-- symbol-observable@1.0.3
+-- typings@1.4.0
| +-- any-promise@1.3.0
| +-- archy@1.0.0
| +-- bluebird@3.4.6
| +-- chalk@1.1.3
| | +-- ansi-styles@2.2.1
| | +-- escape-string-regexp@1.0.5
| | +-- has-ansi@2.0.0
| | | `-- ansi-regex@2.0.0
| | +-- strip-ansi@3.0.1
| | `-- supports-color@2.0.0
| +-- columnify@1.5.4
| | `-- wcwidth@1.0.1
| |   `-- defaults@1.0.3
| |     `-- clone@1.0.2
| +-- has-unicode@2.0.1
| +-- listify@1.0.0
| +-- minimist@1.2.0
| +-- typings-core@1.6.0
| | +-- array-uniq@1.0.3
| | +-- configstore@2.1.0
| | | +-- dot-prop@3.0.0
| | | | `-- is-obj@1.0.1
| | | +-- object-assign@4.1.0
| | | +-- os-tmpdir@1.0.2
| | | +-- osenv@0.1.3
| | | +-- uuid@2.0.3
| | | `-- write-file-atomic@1.2.0
| | |   +-- imurmurhash@0.1.4
| | |   `-- slide@1.1.6
| | +-- debug@2.2.0
| | | `-- ms@0.7.1
| | +-- detect-indent@4.0.0
| | | `-- repeating@2.0.1
| | |   `-- is-finite@1.0.2
| | |     `-- number-is-nan@1.0.1
| | +-- graceful-fs@4.1.9
| | +-- has@1.0.1
| | | `-- function-bind@1.1.0
| | +-- invariant@2.2.1
| | | `-- loose-envify@1.2.0
| | |   `-- js-tokens@1.0.3
| | +-- is-absolute@0.2.5
| | | +-- is-relative@0.2.1
| | | | `-- is-unc-path@0.1.1
| | | |   `-- unc-path-regex@0.1.2
| | | `-- is-windows@0.1.1
| | +-- lockfile@1.0.2
| | +-- make-error-cause@1.2.1
| | | `-- make-error@1.2.1
| | +-- mkdirp@0.5.1
| | | `-- minimist@0.0.8
| | +-- object.pick@1.1.2
| | | `-- isobject@2.1.0
| | |   `-- isarray@1.0.0
| | +-- parse-json@2.2.0
| | | `-- error-ex@1.3.0
| | |   `-- is-arrayish@0.2.1
| | +-- popsicle@8.2.0
| | | +-- arrify@1.0.1
| | | +-- concat-stream@1.5.2
| | | | +-- inherits@2.0.3
| | | | +-- readable-stream@2.0.6
| | | | | +-- core-util-is@1.0.2
| | | | | +-- process-nextick-args@1.0.7
| | | | | +-- string_decoder@0.10.31
| | | | | `-- util-deprecate@1.0.2
| | | | `-- typedarray@0.0.6
| | | +-- form-data@2.1.1
| | | | +-- asynckit@0.4.0
| | | | +-- combined-stream@1.0.5
| | | | | `-- delayed-stream@1.0.0
| | | | `-- mime-types@2.1.12
| | | |   `-- mime-db@1.24.0
| | | +-- throwback@1.1.1
| | | `-- tough-cookie@2.3.1
| | +-- popsicle-proxy-agent@3.0.0
| | | +-- http-proxy-agent@1.0.0
| | | | +-- agent-base@2.0.1
| | | | `-- extend@3.0.0
| | | `-- https-proxy-agent@1.0.0
| | +-- popsicle-retry@3.2.1
| | +-- popsicle-rewrite@1.0.0
| | +-- popsicle-status@2.0.0
| | +-- promise-finally@2.2.1
| | +-- rc@1.1.6
| | | +-- deep-extend@0.4.1
| | | +-- ini@1.3.4
| | | `-- strip-json-comments@1.0.4
| | +-- rimraf@2.5.4
| | | `-- glob@7.1.1
| | |   +-- fs.realpath@1.0.0
| | |   +-- inflight@1.0.6
| | |   | `-- wrappy@1.0.2
| | |   +-- minimatch@3.0.3
| | |   | `-- brace-expansion@1.1.6
| | |   |   +-- balanced-match@0.4.2
| | |   |   `-- concat-map@0.0.1
| | |   +-- once@1.4.0
| | |   `-- path-is-absolute@1.0.1
| | +-- sort-keys@1.1.2
| | | `-- is-plain-obj@1.1.0
| | +-- string-template@1.0.0
| | +-- strip-bom@2.0.0
| | | `-- is-utf8@0.2.1
| | +-- thenify@3.2.1
| | +-- throat@3.0.0
| | +-- touch@1.0.0
| | | `-- nopt@1.0.10
| | |   `-- abbrev@1.0.9
| | +-- typescript@2.0.3
| | `-- zip-object@0.1.0
| +-- update-notifier@1.0.2
| | +-- boxen@0.6.0
| | | +-- ansi-align@1.1.0
| | | +-- camelcase@2.1.1
| | | +-- cli-boxes@1.0.0
| | | +-- filled-array@1.1.0
| | | +-- string-width@1.0.2
| | | | +-- code-point-at@1.0.1
| | | | `-- is-fullwidth-code-point@1.0.0
| | | `-- widest-line@1.0.0
| | +-- is-npm@1.0.0
| | +-- latest-version@2.0.0
| | | `-- package-json@2.4.0
| | |   +-- got@5.6.0
| | |   | +-- create-error-class@3.0.2
| | |   | | `-- capture-stack-trace@1.0.0
| | |   | +-- duplexer2@0.1.4
| | |   | +-- is-redirect@1.0.0
| | |   | +-- is-retry-allowed@1.1.0
| | |   | +-- is-stream@1.1.0
| | |   | +-- lowercase-keys@1.0.0
| | |   | +-- node-status-codes@1.0.0
| | |   | +-- pinkie-promise@2.0.1
| | |   | | `-- pinkie@2.0.4
| | |   | +-- read-all-stream@3.1.0
| | |   | +-- timed-out@2.0.0
| | |   | +-- unzip-response@1.0.1
| | |   | `-- url-parse-lax@1.0.0
| | |   |   `-- prepend-http@1.0.4
| | |   +-- registry-auth-token@3.0.1
| | |   +-- registry-url@3.1.0
| | |   `-- semver@5.3.0
| | +-- lazy-req@1.1.0
| | +-- semver-diff@2.1.0
| | | `-- semver@5.0.3
| | `-- xdg-basedir@2.0.0
| |   `-- os-homedir@1.0.2
| +-- wordwrap@1.0.0
| `-- xtend@4.0.1
`-- zone.js@0.6.21


C:\Users\IEUser\meteor\socially>meteor
[[[[[ C:\Users\IEUser\meteor\socially ]]]]]

=> Started proxy.
=> Started MongoDB.
=> A patch (Meteor 1.4.1.2) for your current release is available!
   Update this project now with 'meteor update --patch'.
=> Errors prevented startup:

   While checking for angular2-compilers@0.6.1:
   error: No compatible binary build found for this package. Contact the package author and ask them to publish it
   for your platform.

=> Your application has errors. Waiting for file change.
```

**Comment by QbyteFennek** [comment]
@abernix Odd, I'll try to dig a bit deeper when I get some more time.

**Comment by abernix** [comment]
Thanks for originally reporting this.  I'm going to close this until you can come up with some further information because it seems to be isolated to your setup **OR** (and this is what I believe to be the problem :smile:) this very likely could be caused by a half-installed version of the Meteor tool, which is very possible due to a CDN problem which was closing connections before the installers/packages were fully downloaded (see #7766).  I hope that issue is resolved/improved now in Meteor 1.4.2 (and with the CDN behaving better).  See my workaround in #7766 if you're having regular problems with this.

Happy to reopen if you come up with additional reproduction steps, and happy to help debug if you continue to run into it.  Cheers!
